.gitmodules

```
@@ -28,3 +28,6 @@
 [submodule "depends/luacov"]
 	path = depends/luacov
 	url = ../../DFHack/luacov.git
+[submodule "depends/googletest"]
+	path = depends/googletest
+	url = https://github.com/google/googletest.git
```

Please use relative URLs to allow for cloning over different protocols:
```suggestion
	url = ../../google/googletest.git
```